sphinx/io.py

```diff
@@ -98,6 +98,11 @@ class SphinxStandaloneReader(SphinxBaseReader):
                   RemoveTranslatableInline, PreserveTranslatableMessages, FilterSystemMessages,
                   RefOnlyBulletListTransform, UnreferencedFootnotesDetector]
 
+    def __init__(self, app, parsers={}, *args, **kwargs):
+        SphinxBaseReader.__init__(self, app, parsers, *args, **kwargs)
+        if app.env.settings['smart_quotes']:
+            self.transforms.append(SphinxSmartQuotes)
```

Comment:
with this, also ``.md`` source files (for example) are in the scope of Smart Quotes, which was not the case before.

Reply:
:-)